partiql-lang/src/main/antlr/PartiQL.g4

```diff
@@ -197,7 +197,7 @@ conflictAction
    [ WHERE <condition> ]
 */
 doReplace
-    : EXCLUDED;
+    : EXCLUDED ( WHERE condition=expr )?;
```

Comment:
Unrelated to this line: should we add a note in the Grammar (or anywhere else applicable) that we can't mix and match the `legacy` with the other?